openpype/hooks/pre_add_last_workfile_arg.py

```diff
@@ -40,5 +42,15 @@ def execute(self):
             self.log.info("Current context does not have any workfile yet.")
             return
 
+        # Determine whether to open workfile post initialization.
+        if self.data["app"].host_name == "maya":
```

Comment:
I believe this can also be `self.host_name` 

See [here](https://github.com/ynput/OpenPype/blob/96719a39b4719dd6d23b638547825bca88fadc7a/openpype/lib/applications.py#L806-L808) - not sure if that'd be better.

Reply:
Good point!